suite/tests/CMakeLists.txt

```diff
@@ -2440,7 +2440,7 @@ if (CLIENT_INTERFACE)
 
       torunonly_ci(tool.reuse ${ci_shared_app} drcachesim
         "reuse_distance.c" # for templatex basename
-        "-ipc_name drtestpipe6 -simulator_type reuse_distance -reuse_distance_threshold 256" "" "")
+        "-ipc_name drtestpipe6 -num_threads 0 -simulator_type reuse_distance -reuse_distance_threshold 256" "" "")
```

Comment:
OK but it is confusing b/c this looks like there's some relationship to the reuse-distance tool: like the reuse-distance tool doesn't support the thread pool or sthg.  I would suggest either a comment explaining, or adding a new test run that looks just an existing run except it has -num_threads 0.